cryostatio/cryostat-operator
# fix(discovery): correct k8s API discovery configuration

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Related to #710

## Description of the change:
Corrects the core container's environment variables.

## Motivation for the change:
Restores the Cryostat instances' ability to do Kubernetes Endpoints discovery.

## How to manually test:
1. Check out, build, push PR as usual
2. Install cert-manager, install operator
3. Create Cryostat CR with cert-manager integration disabled
4. Open web UI and verify that Cryostat can see itself in the target list and Topology view. Interacting with it may not work well due to OpenShift OAuth Proxy redirection bug, but it should appear.
